EstateReporting/Startup.cs

```diff
@@ -131,6 +132,18 @@ public void ConfigureContainer(IServiceCollection services)
 
         private void ConfigureMiddlewareServices(IServiceCollection services)
         {
+            services.AddHealthChecks()
+                    .AddSqlServer(connectionString: ConfigurationReader.GetConnectionString("HealthCheck"),
+                                  healthQuery: "SELECT 1;",
+                                  name: "Read Model Server",
+                                  failureStatus: HealthStatus.Degraded,
+                                  tags: new string[] { "db", "sql", "sqlserver" })
+                    .AddUrlGroup(new Uri($"{ConfigurationReader.GetValue("SecurityConfiguration", "Authority")}/health"),
+                                 name: "Security Service",
+                                 httpMethod: HttpMethod.Get,
+                                 failureStatus: HealthStatus.Unhealthy,
+                                 tags: new string[] { "security", "authorisation" });
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/manual/stuart_ferguson1/EstateReporting/pullRequest?prid=6240712)